src/Standards/Generic/Docs/Arrays/ArrayIndentStandard.xml

```diff
@@ -0,0 +1,26 @@
+<documentation title="Array Indent">
```

Comment:
This file seems out of place in this PR ?

Reply:
How can I add a new documentation?

Reply:
Separate PR ?

Reply:
Yes

Reply:
I just remove the this file in this PR.
So, should I add new documentation in a new PR?